builder/lxc/config.go

```diff
@@ -48,6 +61,10 @@ func NewConfig(raws ...interface{}) (*Config, error) {
 		c.OutputDir = fmt.Sprintf("output-%s", c.PackerBuildName)
 	}
 
+	if c.ExportName == "" {
+		c.ExportName = "rootfs.tar.gz"
```

Comment:
@mhan do we want this as an option ? Either way we may need to keep that if statement to avoid this `"--exclude=./rootfs/dev/log"` when we export specific folders.